FirebaseStorage/Tests/Integration/FIRStorageIntegrationTests.m

```diff
@@ -23,32 +24,37 @@
 /**
  * Firebase Storage Integration tests
  *
- * To run these tests, you need to define the following access rights for your Firebase App:
- * - unauthentication read/write access to /ios/public
- * - authentication read/write access to /ios/private
+ * To run these tests, you need to define the following access rights:
  *
- * A sample configuration may look like:
+  rules_version = '2';
+  service firebase.storage {
+    match /b/{bucket}/o {
+      match /ios {
+        match /public/{allPaths=**} {
+          allow write: if request.auth != null;
+          allow read: if true;
+        }
+        match /private/{allPaths=**} {
+          allow read, write: if false;
+        }
+      }
+    }
+  }
  *
- * rules_version = '2';
- * service firebase.storage {
- *   match /b/{bucket}/o {
- *     ...
- *     match /ios {
- *       match /public/{allPaths=**} {
- *         allow read, write;
- *       }
- *       match /private/{allPaths=**} {
- *         allow none;
- *       }
- *     }
- *   }
- * }
+ * You also need to enable email/password sign in and add a test user in your
+ * Firebase Authentication settings. Your account credentials need to match
+ * the credentials defined in `kTestUser` and `kTestPassword`.
  *
  * You can define these access rights in the Firebase Console of your project.
  */
+
+NSString *const kTestUser = @"test@example.com";
+NSString *const kTestPassword = @"testing";
```

Comment:
BTW are public credentials an issue from b/155892305 perspective? We can move credentials to secrets (e.g. like Auth do).

Reply:
We are planning to do so as well.